templates/entry/comment/view.html.twig

```diff
@@ -0,0 +1,56 @@
+{%- set V_TRUE = constant('App\\Controller\\User\\ThemeSettingsController::TRUE') -%}
+{%- set V_FALSE = constant('App\\Controller\\User\\ThemeSettingsController::FALSE') -%}
+
+{%- set DYNAMIC_LISTS = app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_DYNAMIC_LISTS'), V_FALSE) -%}
+{%- set SHOW_COMMENT_USER_AVATARS = app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_COMMENTS_SHOW_USER_AVATAR'), V_TRUE) -%}
+
+{% extends 'base.html.twig' %}
+
+{%- block title -%}
+    {{- 'comments'|trans }} - {{ entry.title }} - {{ parent() -}}
+{%- endblock -%}
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    <style>
+        #entry-comment-{{comment.id}} {
+            border-top: var(--kbin-alert-danger-border);
+            border-bottom: var(--kbin-alert-danger-border);
+            border-right: var(--kbin-alert-danger-border);
+        }
+    </style>
```

Comment:
this is a bit hacky where; I can't modify the css in the nested comments (like adding `highlighted` to the comment class list) as it's cached html, so instead this sets the css for the comment that is being viewed to style it differently. It could also be done via js but this doesn't seem _that_ bad,  but I could be wrong